Resources/Private/Partials/Bootstrap/Post/Form.html

```diff
@@ -1,5 +1,16 @@
 {namespace mmf=Mittwald\Typo3Forum\ViewHelpers}
 {namespace b=Mittwald\Typo3Forum\ViewHelpers\Bootstrap}
+<script type="text/javascript">
+	$(document).ready(function() {
+	function validate() {
+		var file_size = $('#att')[0].files[0].size / 1000;
+		if (file_size > 2000) {
+			alert("Uploaded file(s) size should be less than 2MB");
```

Comment:
Why to MB? That's a limit that should be looked up in php.ini and could be 200MB too.

Reply:
Limit in the extension (or of TYPO3) is thinkable and useful additionally I think, if it's not too much work.

Reply:
Also see #44